docs/build/eps.md

```diff
@@ -142,58 +142,70 @@ Dockerfile instructions are available [here](https://github.com/microsoft/onnxru
 
-    ```bash
-    export CUDACXX="/usr/local/cuda/bin/nvcc"
+      1. Check [this official blog](https://developer.nvidia.com/blog/simplifying-cuda-upgrades-for-nvidia-jetson-users/) for CUDA 11.8 upgrade instruction.
 
-    ```
+      2. CUDA 12.x might only be available to Jetson Orin and newer series. 
 
-    or:
+   2. CMake can't automatically find the correct `nvcc` if it's not in the `PATH`. `nvcc` can be added to `PATH` via:
 
-    ```bash
-    export PATH="/usr/local/cuda/bin:${PATH}"
-    ```
+      ```bash
+      export PATH="/usr/local/cuda/bin:${PATH}"
+      ```
 
-3. Install the ONNX Runtime build dependencies on the Jetpack 4.6.1 host:
+       or:
 
-    ```bash
-    sudo apt install -y --no-install-recommends \
-      build-essential software-properties-common libopenblas-dev \
-      libpython3.6-dev python3-pip python3-dev python3-setuptools python3-wheel
-    ```
+      ```bash
+      export CUDACXX="/usr/local/cuda/bin/nvcc"
+      ```
 
-4. Cmake is needed to build ONNX Runtime. Because the minimum required version is 3.18,
-   it is necessary to build CMake from source. Download Unix/Linux sources from https://cmake.org/download/
-   and follow https://cmake.org/install/ to build from source. Version 3.23.0 has been tested on Jetson.
+3. Install the ONNX Runtime build dependencies on the Jetpack 5.1.2 host:
 
-5. Build the ONNX Runtime Python wheel:
+   ```bash
+   sudo apt install -y --no-install-recommends \
+     build-essential software-properties-common libopenblas-dev \
+     libpython3.8-dev python3-pip python3-dev python3-setuptools python3-wheel
+   ```
 
-    ```bash
-    ./build.sh --config Release --update --build --parallel --build_wheel \
-    --use_cuda --cuda_home /usr/local/cuda --cudnn_home /usr/lib/aarch64-linux-gnu
-    ```
+4. Cmake is needed to build ONNX Runtime. For ONNX Runtime 1.16, the minimum required CMake version is 3.26 (version 3.27.4 has been tested). This can be either installed by:
 
-    Note: You may optionally build with TensorRT support.
+   1. (Unix/Linux) Build from source. Download sources from [https://cmake.org/download/](https://cmake.org/download/)
+      and follow [https://cmake.org/install/](https://cmake.org/install/) to build from source. 
+   2. (Ubuntu) Install deb package via apt repository: e.g [https://apt.kitware.com/](https://apt.kitware.com/)
 
-    ```bash
-    ./build.sh --config Release --update --build --parallel --build_wheel \
-    --use_tensorrt --cuda_home /usr/local/cuda --cudnn_home /usr/lib/aarch64-linux-gnu \
-    --tensorrt_home /usr/lib/aarch64-linux-gnu
-    ```
+5. Build the ONNX Runtime Python wheel (update path to CUDA/CUDNN/TensorRT libraries if necessary):
 
----
+   1. Build `onnxruntime-gpu` wheel with CUDA support:
```

Comment:
i think we should remove this section.
Only provide example where both CUDA and TensorRT EP are both enabled
because that is what the published ort gpu packages include.